backend/.kubernetes/kubernetes.yaml

```diff
@@ -0,0 +1,175 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: spellbook
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: spellbook
+  name: spellbook-api
+  labels:
+    app: spellbook-api
+spec:
+  selector:
+    matchLabels:
+      app: spellbook-api
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: spellbook-api
+    spec:
+      containers:
+        - name: spellbook-api-app
+          image: 083767677168.dkr.ecr.us-east-2.amazonaws.com/spellbook-prod-ecr
+          ports:
+            - containerPort: 80
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 80
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            timeoutSeconds: 5
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 80
+            initialDelaySeconds: 40
+            periodSeconds: 20
+            timeoutSeconds: 5
+          env:
+            - name: SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: api-secrets
+                  key: django-secret
+            - name: KUBE_SQL_ENGINE
+              valueFrom:
+                secretKeyRef:
+                  name: api-secrets
+                  key: db-engine
+            - name: KUBE_SQL_DATABASE
+              valueFrom:
+                secretKeyRef:
+                  name: api-secrets
+                  key: db-name
+            - name: KUBE_SQL_USER
+              valueFrom:
+                secretKeyRef:
+                  name: api-secrets
+                  key: db-user
+            - name: KUBE_SQL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: api-secrets
+                  key: db-password
+            - name: KUBE_SQL_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: api-secrets
+                  key: db-host
+            - name: KUBE_SQL_PORT
+              valueFrom:
+                secretKeyRef:
+                  name: api-secrets
+                  key: db-port
+
+
+---
+# Autoscaling of the api pods
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: spellbook-api-autoscaler
+  namespace: spellbook
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: spellbook-api
+  minReplicas: 3
+  maxReplicas: 36
+  targetCPUUtilizationPercentage: 80
+
+---
+# API Service
+apiVersion: v1
+kind: Service
+metadata:
+  name: spellbook-api-service
+  namespace: spellbook
+  labels:
+    app: spellbook-api
+    tier: web
+spec:
+  type: NodePort
+  selector:
+    app: spellbook-api
+  ports:
+    - port: 80
+      targetPort: 80
+
+---
+# Ingress Load Balancer
+apiVersion: networking.k8s.io/v1
```

Comment:
If this load balancer replaces nginx, whos i in charge of serving static files?

Reply:
There is now an nginx running on the api image itself as well, with the same configuration for static files as you had before. So for now that will serve the static files. We could entertain the notion of instead pushing the files to their own CDN as part of our build process and have the CDN redirect cdn routes there. However, I think long term we wanted to offload the form work from the django admin panel to a form on the client, so I'm happy with it the way it is for now.

Reply:
I can understand your poiny, but I must say that through my experience with Django admin it would be very, very expensive to trying to replicate all those features I added on the client. Surely we will have forms, one day, for users to subit their combo suggestions from the client. But I admit I'm doubting we will ever replace the admin dashboard.

If you agree, I'd think of a simpler, cleaner solution to the problem, and I'm quoting my last discord message:
>  For instance, I think that django itself could serve those or we could even have a single, separate nginx instance that does that maybe